config/application.rb

```diff
@@ -79,7 +79,7 @@
   # local testing environment you may want to log these
   # messages to 'test.log' file to keep your
   # testing suite output readable.
-  # config.logger = Jets::Logger.new($strerr)
+  config.logger = Jets::Logger.new(STDOUT)
```

Comment:
How will this effect logging in the aws environment. The comment above

> By default logger needs to log to $stderr for CloudWatch to receive Lambda messages

makes me concerned about changing the logging to stdout.

Reply:
updating from **STDOUT** to **$stderr**, that make sense. 
thanks